pointcloud/voxel.go

```diff
@@ -424,7 +424,7 @@ func (vg VoxelGrid) GetNNearestVoxels(v *Voxel, n uint) []VoxelCoords {
 	return neighborKeys
 }
 
-// ConvertToPointCloudWithValue converts the voxel grid to a point cloud with values
+// ConvertToPointCloudWithValue converts the voxel grid to a point cloud with
 // values are containing the labels.
```

Comment:
[nit] So it catches duplicated words, but is there any way to actually just spell/grammar check comments (especially godoc ones)?

Reply:
spell is, grammar idk